pandera/pandera.py

```diff
@@ -229,22 +249,27 @@ def prepare_series_input(self, series, dataframe):
 
         return self._format_input(groupby_obj, self.groups)
 
-    def prepare_dataframe_input(self, dataframe):
+    def prepare_dataframe_input(self,
+                                dataframe: pd.DataFrame):
         """Prepare input for DataFrameSchema check."""
         if self.groupby is None:
             return dataframe
         else:
             groupby_obj = dataframe.groupby(self.groupby)
         return self._format_input(groupby_obj, self.groups)
 
-    def _vectorized_check(self, parent_schema, check_index, check_obj):
+    def _vectorized_check(self,
+                          parent_schema,
+                          check_index: int,
+                          check_obj: Dict[str, pd.Series]):
         """Perform a vectorized check on a series.
 
-        :param parent_schema: The schema object that is being checked and that
-            was inherited from the parent class.
-        :param check_index: The validator to check the series for
-        :param dict check_obj: a dictionary of pd.Series to be used by
+        :param DataFrameSchema parent_schema: The schema object that is
```

Comment:
`Union[DataFrameSchema, SeriesSchemaBase]`

Reply:
will resolve after pandera is split into submodules